lib/src/main/java/io/ably/lib/realtime/Presence.java

```diff
@@ -511,6 +546,14 @@ public void updatePresence(PresenceMessage msg, CompletionListener listener) thr
 		}
 	}
 
+	/**
+	 * (RTP13) Presence#syncComplete returns true if the initial SYNC operation has completed for
+	 * the members present on the channel
+	 */
+	public boolean syncComplete() {
```

Comment:
I don't think it's a method, RTP13 states `Presence#syncComplete returns true..` which does not mean it's a method, the spec is language agnostic.

More importantly though, in the IDL, it's defined as:
`syncComplete: Bool` which is an attribute